CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -282,9 +229,67 @@ def add_item(self) -> DB_Response:
     def get_fridge_locations(self):
         pass
 
-class FrigeCheckIn(DB_Item):
-    def __init__(self, db_client: "botocore.client.DynamoDB"):
+
+class FridgeReport(DB_Item):
+
+    REQUIRED_FIELDS = ['fridge_username', 'status', 'notes', 'fridge_percentage']
+    ITEM_TYPES = {
+        'notes': 'S',
+        'fridge_username': 'S',
+        'image_url': 'S',
+        'timestamp': 'N',
+        'status': 'S',
+        'fridge_percentage': 'S'
+    }
+    TABLE_NAME = "fridge_report"
+    VALID_STATUS = set(["working", "needs cleaning", "needs servicing", "not at location"])
+    VALID_FRIDGE_PERCENTAGE = set(["0", "33", "66", "100"])
+    
+    def __init__(self, db_client: 'botocore.client.DynamoDB', fridge_report: dict=None):
         super().__init__(db_client=db_client)
+        if fridge_report is not None:
+            self.notes:str = fridge_report.get('notes', None)
+            self.status:str = fridge_report.get('status', None)
+            self.image_url:str = fridge_report.get('image_url', None)
+            self.fridge_username:str = fridge_report.get('fridge_username', None)
+            self.fridge_percentage:int = fridge_report.get('fridge_percentage', None) 
+    
+    def set_timestamp(self):
+        self.timestamp = str(int(time.time()))
```

Comment:
nit: have we settled on the time format we're using?

Reply:
So we will be storing things in UTC timezone. 

In dynamodb we will be using epoch time (better for filtering/querying)

But our clients will get this format: "2022-03-29T18:10:38.547Z"

Reply:
Okay, just wanted to confirm the epoch idea. 
1. `int(time.time())` gives the epoch time?
2. This epoch time will be localized, or will it be in UTC? (I think it might be localized though)

Reply:
epoch time is UTC https://docs.python.org/3/library/time.html